README.md

```diff
@@ -66,7 +66,13 @@ Note: The version tested with Zivid cameras is 1.8.1.
 
        pip install -r requirements.txt
 
-6. Open and run one of the samples.
+6. Add the directory `source` to PYTHONPATH
```

Comment:
This needs to be done from the root, e.g. C:\Users\SatjaSivcev\Documents\python-samples otherwise it doesn't work

Reply:
done

Reply:
PWD will give absolute path

Reply:
It didn't work for me. I will test it again then.